src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,209 @@
+     */
+    public static void deleteTaskMsg(int index, int newSizeOfTasks, Task taskToDelete) {
+        String outputMsg = "___________________________________________________________"
+                + "\n (ಠ‿↼)"
+                + "\n Noted. This task has been removed from your task list:"
+                + "\n      Task #" + index + ". " + taskToDelete
+                + "\n Now you have " + newSizeOfTasks + " tasks in the list."
+                + "\n___________________________________________________________\n";
+        System.out.println(outputMsg);
+    }
+
+    /**
+     * Prints out all tasks stored in the task list.
+     *
+     * @param tasks The overall user's task list.
+     */
+    public static void getAllTasksMsg(ArrayList<Task> tasks) {
+        String outputMsg = "";
+        if (tasks.isEmpty()) {
+            outputMsg = "___________________________________________________________"
+                    + "\n (⊙ ‿ ⊙)"
+                    + "\n Task list is empty, please try to add some tasks first."
+                    + "\n___________________________________________________________\n";
+        } else {
+            outputMsg = "___________________________________________________________"
+                    + "\n (⊙ ‿ ⊙)"
+                    + "\n You have " + tasks.size() + " tasks in total."
+                    + "\n Here they are:";
+            for (int i = 0; i < tasks.size(); i++) {
+                outputMsg += "\n      " + (i + 1) + ". " + tasks.get(i);
+            }
+            outputMsg += "\n___________________________________________________________\n";
+        }
+
+        System.out.println(outputMsg);
+    }
+
+    /**
+     * Prints out all tasks that match the keyword stored in the task list.
+     *
+     * @param keyword A string of keyword to find the matching tasks.
+     * @param matchedTasks An ArrayList of all the tasks that match with the keyword.
+     */
+    public static void findMatchingTasks(String keyword, ArrayList<Task> matchedTasks) {
+        String outputMsg = "";
+        if (matchedTasks.isEmpty()) {
+            outputMsg = "___________________________________________________________"
+                    + "\n (⊙ ‿ ⊙)"
+                    + "\n There is no matching task with the keyword - '" + keyword + "'."
+                    + "\n___________________________________________________________\n";
+        } else {
+            outputMsg = "___________________________________________________________"
+                    + "\n (⊙ ‿ ⊙)"
+                    + "\n You have " + matchedTasks.size() + " tasks matched with keyword - '"
+                    + keyword + "'."
+                    + "\n Here they are:";
+            for (int i = 0; i < matchedTasks.size(); i++) {
+                outputMsg += "\n      " + (i + 1) + ". " + matchedTasks.get(i);
+            }
+            outputMsg += "\n___________________________________________________________\n";
+        }
```

Comment:
Might want to use at `String.format()` to make it cleaner

Reply:
That's true, thank you!